api/migrations/000003_create_nft_table.up.sql

```diff
@@ -0,0 +1,18 @@
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS nft (
+    id uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    collection_address VARCHAR NOT NULL,
+    token_id INTEGER NOT NULL,
+    owner_address VARCHAR,
+    image_url VARCHAR,
+    traits VARCHAR,
+    backstory VARCHAR(3000),
```

Comment:
I am thinking we may also need to save backstory on other decentralized storage, like IPFS, or Arweave. do you need to add a link to the backstory?

Reply:
Do we want to use decentralized storage for this project? We can easily add a new column if we need it later

Reply:
we can add later, please just create a github issue then.